fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/query/WeightEvaluationQuery.java

```diff
@@ -185,13 +183,18 @@ private static DataSource makeInnerQuery(DruidAggregationQuery<?> query, double
         DruidAggregationQuery<?> innerQuery = query.getInnermostQuery();
 
         List<Aggregation> aggregations;
-        aggregations = Collections.<Aggregation>singletonList(new CountAggregation("ignored"));
+        aggregations = Collections.singletonList(new CountAggregation("ignored"));
 
         // Get the inner post aggregation
         List<PostAggregation> postAggregations;
-        postAggregations = Collections.<PostAggregation>singletonList(new ConstantPostAggregation("count", weight));
+        postAggregations = Collections.singletonList(new ConstantPostAggregation("count", weight));
+
+        if (!(innerQuery.getQueryType() instanceof DefaultQueryType)) {
```

Comment:
We should add a note (likely to the changelog) that this test extension point doesn't support new types of queries.